### ISSUE TITLE
Animated Gif appears as static image in sunlit timeline

### ISSUE THREAD

**Issue by adamprocter** [microdotblog/sunlit#98]
Should these appear as animations in Sunlit?
Have play icon like videos ?

**Comment by manton** [comment]
Yeah, animated GIFs aren't supported yet. They probably should be, at least for playback if not posting.

**Comment by cheesemaker** [comment]
If we do this, it should be part of 3.4 and the non-Micro.blog support because I'm not sure Micro.blog itself will ever support these
